sound/soc/intel/boards/sof_sdw.c

```diff
@@ -599,6 +599,42 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		},
 		.dai_num = 1,
 	},
+	{
+		.part_id = 0x712,
+		.version_id = 3,
+		.dais =	{
+			{
+				.direction = {true, true},
+				.dai_name = "rt712-sdca-aif1",
+				.dai_type = SOF_SDW_DAI_TYPE_JACK,
+				.dailink = {SDW_JACK_OUT_DAI_ID, SDW_JACK_IN_DAI_ID},
+				.init = sof_sdw_rt712_sdca_init,
+				.exit = sof_sdw_rt712_sdca_exit,
+			},
+			{
+				.direction = {true, false},
+				.dai_name = "rt712-sdca-aif2",
+				.dai_type = SOF_SDW_DAI_TYPE_AMP,
+				.dailink = {SDW_AMP_OUT_DAI_ID, SDW_AMP_IN_DAI_ID},
```

Comment:
.dailink = {SDW_AMP_OUT_DAI_ID, SDW_UNUSED_DAI_ID} ?

if the direction for capture is false, why would we have an AMP_IN_DAI_ID?

Reply:
> .dailink = {SDW_AMP_OUT_DAI_ID, SDW_UNUSED_DAI_ID} ?
> 
> if the direction for capture is false, why would we have an AMP_IN_DAI_ID?

Right, replaced with SDW_UNUSED_DAI_ID